package.json

```diff
@@ -4,7 +4,10 @@
   "description": "JSON Schema to JSDoc generator",
   "main": "index.js",
   "scripts": {
-    "test": "nyc mocha --require chai/register-assert",
+    "lint": "eslint .",
+    "mocha": "mocha --require chai/register-assert",
+    "nyc": "nyc npm run mocha",
+    "test": "npm run lint && npm run nyc",
```

Comment:
Looking at this makes me want to move to Jest

Reply:
If it is only because there are too many scripts--I personally like to be able to ensure that `test`, e.g., when run through CI, is doing linting, but that when doing rapid debugging, including possibly adding console logging which might get reported for linting, one can run a script which doesn't spend extra time checking linting.

And though I think checking coverage is good for CI (esp. if you are ok setting thresholds to 100% after my testing PR) as well as good at times for local testing, especially when adding `only` to a test to figure out a problem unrelated to coverage, I don't like the extra time it takes for coverage to run (and the extra messages it gives).

To get all this behavior, one would need the same number of scripts with `jest` as well anyways.

But if you want jest for some other reason (e.g., so as to be able to use the same script name, i.e., `jest` and `jest --coverage`), or if you don't want 3 separate scripts, I can of course change.

Reply:
The some other reason is not just about package.json scripts. Jest has other benefits over mocha + chai.

Here's how I would do it:
```
"lint": "eslint .",
"test": "jest"
```
Then if  you want it in CI, it can `npm run lint && npm t` . Checking coverage locally? `npm t -- --coverage`

Reply:
Ok, sure. I also added an `.npmignore` file to avoid any coverage or doc files getting published.